builder/oracle/classic/config.go

```diff
@@ -23,9 +23,13 @@ type Config struct {
 	apiEndpointURL *url.URL
 
 	// Image
-	ImageName string `mapstructure:"image_name"`
-	Shape     string `mapstructure:"shape"`
-	ImageList string `mapstructure:"image_list"`
+	ImageName       string `mapstructure:"image_name"`
+	Shape           string `mapstructure:"shape"`
+	SourceImageList string `mapstructure:"source_image_list"`
+	DestImageList   string `mapstructure:"dest_image_list"`
+	// Optional; if you don't enter anything, the image list description
+	// will read "Packer-built image list"
+	DestImageListDescription string `mapstructure:"dest_image_list_description`
```

Comment:
I think this attribute could be named just `image_description` - `dest_image_list_description` seems overly long